### PR TITLE
Fixes display issues in the tablesheet.

### PR DESCRIPTION
**Pull request by EstherDarkish**
Fixes issues in Gridstudy's Spreadsheet :
- No longer possible to "sort" by the edit column (the one with pen icons)
- Fixes display of edit boxes for TapChangerSelector and NumericalField
- Adds a line on the bottom of the table. It looks better, and prevents a cropping effect when editing the last line

### CODE REVIEW

File: src/components/network/equipment-table.js [hunk 1]
```diff
@@ -20,6 +32,20 @@ export const EquipmentTable = (props) => {
         }
     }, [props.columns]);
 
+    const lastLineCellRender = useCallback(
+        (columnDefinition, key, style) => {
+            if (columnDefinition.editColumn) {
```
Comment (laurgarn) [inline src/components/network/equipment-table.js:37]: May be something like :
```
<div key={key} style={style}>
    {columnDefinition.editColumn && <div className={classes.emptyLastLineCell}></div>}
</div>
```

Comment (EstherDarkish) [reply src/components/network/equipment-table.js:37]: Yes thank you, that's way better.

File: src/components/network/network-table.js [hunk 2]
```diff
@@ -579,14 +574,16 @@ const NetworkTable = (props) => {
...
                 >
-                    <div
-                        className={clsx(classes.tableCell, {
-                            [classes.valueInvalid]:
-                                columnDefinition.canBeInvalidated &&
-                                props.loadFlowStatus !== RunningStatus.SUCCEED,
-                        })}
-                    >
-                        <OverflowableText text={text} />
+                    <div className={clsx(classes.tableCell)}>
```
[the remark this answers is not in the document]
Comment (EstherDarkish) [reply src/components/network/network-table.js:577]: Good catch

File: src/components/network/equipment-table.js [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
 import {
     MIN_COLUMN_WIDTH,
     HEADER_ROW_HEIGHT,
     ROW_HEIGHT,
 } from './config-tables';
+import makeStyles from '@mui/styles/makeStyles';
+
+const useStyles = makeStyles(() => ({
+    emptyLastLineCell: {
+        margin: '5px',
```
Comment (sBouzols) [inline src/components/network/equipment-table.js:13]: use theme.spacing(1) or 2, or 3 ... not raw px value

Comment (EstherDarkish) [reply src/components/network/equipment-table.js]: Ok done.

**Pull request merged**